crates/sovereign-sdk/rollup-interface/src/node/telemetry.rs

```diff
@@ -0,0 +1,18 @@
+//! Provide the data structure for telemetry targets
```

Comment:
To not import anything in rollup-interface

Reply:
The types defined here are returned by the new method defined in `DaService`. This trait lives in rollup interface which cannot import primitives or common because both of the mentioned crates import rollup-interface. This is preventing cyclic dependencies. 

TBH, i was thinking that we merge rollup interface with our own primitives into a single crate.

Reply:
> TBH, i was thinking that we merge rollup interface with our own primitives into a single crate.

That also would be nice